bin/init.bash

```diff
@@ -278,6 +294,8 @@ set_nginx_config() {
     replace_set_me "${file}" '.ingressNginx.controller.service.enabled' "${service_enabled}"
     replace_set_me "${file}" '.networkPolicies.global.externalLoadBalancer' "${external_load_balancer}"
     replace_set_me "${file}" '.networkPolicies.global.ingressUsingHostNetwork' "${ingress_using_host_network}"
+    replace_set_me "${file}" '.ingressNginx.controller.service.allocateLoadBalancerNodePorts' 'true'
```

Comment:
If this is always set to true, why not just set it to true in the default config instead?

Reply:
It can, but also thought it's better to have it there so that if time comes for it to be changed, it is clearer to look for and configure it there (based on experience with the dns setting)